projects/plugins/jetpack/_inc/client/traffic/seo.jsx

```diff
@@ -74,8 +75,16 @@ export const SEO = withModuleSettingsFormHelpers(
 		};
 
 		toggleSeoEnhancer = () => {
+			const isEnabled = this.props.getOptionValue( 'ai_seo_enhancer_enabled' );
+
+			analytics.tracks.recordEvent( 'jetpack_wpa_settings_toggle', {
+				module: 'seo-tools',
+				setting: 'ai_seo_enhancer_enabled',
+				toggled: ! isEnabled ? 'on' : 'off',
+			} );
+
 			this.props.updateOptions( {
-				ai_seo_enhancer_enabled: ! this.props.getOptionValue( 'ai_seo_enhancer_enabled' ),
+				ai_seo_enhancer_enabled: ! isEnabled,
```

Comment:
Not asking for a change, just an opinion: it is easier to read without the negation. Something like `isEnabling = ! isEnabled`, and then using `isEnabling` on both places.